lms/djangoapps/courseware/date_summary.py

```diff
@@ -353,13 +359,17 @@ def date(self, date):
     def title(self):
         return self.assignment_title
 
+    @property
+    def title_html(self):
+        return self.assignment_title_html
+
     def set_title(self, title, link=None):
```

Comment:
Do you need to declare this as a setter?

Reply:
I don't I can. If it was the standard property setter, it would just be something like `Obj.title = 'blah'`, but since we want the kwarg, it just needs to be a method.